ovip-0011.md

```diff
@@ -0,0 +1,71 @@
+```pseudocode
+OVIP: 11
+Title: VASP Registries
+Author: David Riegelnig <david.riegelnig@bitcoinsuisse.com>
+Discussions-To: https://community.openvasp.org/#narrow/stream/21-protocol-.2F.20ovip
+Status: Proposal
+Type: Standard
+Created: 2020-06-11
+```
+
+# Abstract
+
+Virtual Assets Service Providers (VASPs) using the OpenVASP protocol establish their identity based on a self-assigned VASP Identifier whose uniqueness is ensured via the Index Contract.
+
+The holder of the VASP Identifier can prove his authenticity as well as ownership of the respective VASP Contract by signing with the private key (see [ovip-0003]()).
+
+Those sending an OpenVASP Message to the holder of a specific VASP Identifier can therefore be ensured that only this holder will be able to receive and decrypt the Message when using the public keys (for transport and session layer) available from the respective VASP Contract for encryption.
+
+This approach in combination with an identity-based routing (such as Ethereum Whisper) implements the basic principle that no registration with any central party is required to securely route messages and use the protocol on a technical level.
+
+However, having full confidence in the link between the VASP Identifier and a real-world entity requires additional assurance, either through first-hand evidence or verification by a trusted third party. 
+
+The **VASP Registry** specification in this document defines a smart contract interface for how trusted third parties can provide a registry of claims (assertions) about VASPs via the VASP Identifier.
+
+# Definitions
+
+**VASP Identifier**</br>
+Globally unique VASP identifier as specified in [ovip-0002]().
+
+**VASP Registry**</br>
+Smart contract operated by a trusted third party providing access to claims about a VASP referenced by its *VASP Identifier*.
+
+**Credential**</br>
+Claim about a VASP issued by a trusted third party via its *VASP Registry*.
+
+# Specification
+
+## 1. Contract Interface
+
+Conformant *VASP Registries* should implement the following function:
+
+```pseudocode
+getCredentialRef(vaspID:bytes12): bytes32 {public}
```

Comment:
string is more appropriate here, as also a url or any other method of linking to the content might be used

Reply:
We should also return the hash of the Credential Content. We will then decouple authenticity of the content from it's location